components/icons/Contributing.js

```diff
@@ -0,0 +1,25 @@
+export default function IconContributing ({ className }) {
```

Comment:
The icons added for the menu should not be added as the SVG code inside components. All the illustrations used should be added inside `/public/img/illustrations/icons/` folder and not in the form of code, but in the SVG format. Remove this `icons` folder inside the `components` folder and place the files under the right directory. Also, don't forget to change the path of icons in the `CommunityItems.js`.

Reply:
For the icons, I met them in the `components/icons` folder. I wasn't the one that created them there. So I'm not sure I will want to change the location since I didn't make the folder structure.

Reply:
Actually, the `components/icons` folder has been made to structures some special types of icons like GitHub.js, Modelina.js, Parser.js, etc. whereas the type of SVG icons are directly added to the `/public/img/illustrations/icons/` in order to use it. So, it will be better you directly export the SVG icons to the above directory and then use it. 

I also ask @magicmatatjahu here if the present addition of icons are correct or it should be under `public` folder.

Reply:
Please do ask.